source/utilities.tex

```diff
@@ -16018,8 +16026,9 @@
 then \placeholder{C} is appended unchanged.
 \end{itemize}
 
-%% FIXME: Example is incomplete; s2 and s6 are missing below;
+%% FIXME: Example is incomplete; s2 and s6 from P2286R8 are missing below;
 %% FIXME: their Unicode characters are not available in our font (Latin Modern).
+%% FIXME: ... same for s8 from P2713R1
```

Comment:
But you're adding an `s6`. Considering that the paper doesn't have `s2` either, the intention must have been to continue the sequence.

Reply:
I sympathize with what you're trying to do here, but we don't want %%FIXMEs to appear in the final printed document, so unfortunately we can't put in placeholders for these missing examples as you suggested. My way of handling this was to just amend the old FIXME comment to say which s8 we were talking about (the one from the original paper). If that s8 is ever added back, it will become sN where N is the next in the sequence.

Reply:
> If that s8 is ever added back, it will become sN where N is the next in the sequence.

There might be value in retaining the would-have-been order if the FIXMEs weren't necessary.

Reply:
Agreed, especially if it would keep similar examples together. That said, we can always renumber the variables to insert the missing examples. As it is, it looks odd to have s2 skipped in the final doc.

Reply:
That's definitely true.

Reply:
The CD has examples `s2` and `s6`, and the DIS will, too.